token/services/db/sql/common/tokens.go

```diff
@@ -1010,17 +1044,18 @@ func (t *TokenTransaction) Delete(ctx context.Context, txID string, index uint64
 	return nil
 }
 
-func (t *TokenTransaction) StoreToken(ctx context.Context, tr driver.TokenRecord, owners []string) error {
```

Comment:
This will not capture anymore the events under the same span, when we create a new context for the token transaction.

```

 tx := NewTokenTransaction(ctx)
 ...
func MyFunc1(tx *TokenTransaction) {
 span, newCtx := tracer.NewSpanFromCtx(ctx, ...)
span.AddEvent("one")

tx.Delete(...)
}

func MyFunc2(tx *TokenTransaction) {
 span, newCtx := tracer.NewSpanFromCtx(ctx, ...)
span.AddEvent("two")

tx.Add(...)
}
```